### DISCUSSION TITLE
Alternative to The Signal Description Language

### DISCUSSION THREAD

**Discussion by cahoy** [gusmanb/logicanalyzer#56] [Ideas]
First of all, BIG THANKS for this project! Love it.

After reading your documentation, it seems that you're inventing a new signal description language to handle the data in the background.

I would suggest to use a description language that is already established in the industry and actively maintained. One standard that I would recommend is IEEE Std 1450.1-2005 (STIL). It is used to represent digital waveforms and the language that you described is comparable to the STIL document chapter 15 about Pattern Data. There are other standards like WGL, VCD, but I'm less familiar with those older standards.

STIL format is widely used in IC development industry, especially in the design & test area. These guys are the ones that could directly benefit from your project, since they often need to have access to a logic analyzer to verify their IC in the lab. And I think using an established standard would open opportunities to extend the project.

**Comment by gusmanb** [comment]
Hi!

Thanks for the suggestion. I will take a look at it and if I see that it can be implemented I will try to :)

Cheers!
